includes/learnopengl/model.h

```diff
@@ -173,6 +224,42 @@ class Model
     vector<Texture> loadMaterialTextures(aiMaterial *mat, aiTextureType type, string typeName)
     {
         vector<Texture> textures;
+        
+        // First check if there isn't a texture, if so create an empty image
+        if (mat->GetTextureCount(type) == 0)
+        {
+            bool skip = false;
+            for (unsigned int j = 0; j < textures_loaded.size(); j++)
+            {
+                std::string name = "DEFAULT_" + typeName;
+                if (std::strcmp(textures_loaded[j].path.data(), name.c_str()) == 0)
```

Comment:
You can do this:
if(textures_loaded[j].path == "DEFAULT_" + typeName)

No need to create temporary copy! Because you have const& string! :)

Reply:
Aha! I didn't know you could do that! 👍